internal/engine/image_build_and_deployer_test.go

```diff
@@ -123,6 +123,38 @@ func TestDeployPodWithMultipleImages(t *testing.T) {
 		"Expected image to appear once in YAML: %s", f.k8s.Yaml)
 }
 
+func TestDeployPodWithMultipleFastBuildImages(t *testing.T) {
```

Comment:
since we're moving away from FastBuild, might be more useful to test this behavior with LiveUpdate instead